pom.xml

```diff
@@ -29,7 +29,7 @@
     <repositories>
         <repository>
             <id>Eclipse</id>
-            <url>http://download.eclipse.org/releases/2021-09</url>
+            <url>http://download.eclipse.org/releases/2021-12</url>
```

Comment:
Might it not be a good idea to switch to 2022-03? Unless there are some bugs, which could prevent this...

Reply:
Done. Let's look whether it builds :)